tests/foreman/ui/test_hostcollection.py

```diff
@@ -626,3 +627,24 @@ def test_positive_remove_package_group(self):
             for package in FAKE_0_CUSTOM_PACKAGE_GROUP:
                 self._validate_package_installed(
                     self.hosts, package, expected_installed=False)
+
+    @tier3
+    def test_positive_install_errata(self):
```

Comment:
Probably worth mentioning - `FAKE_2_CUSTOM_PACKAGE` is the latter version of `FAKE_1_CUSTOM_PACKAGE`, so by verifying the latest version of package is installed we're actually verifying errata was installed successfully.